tests/saitests/switch.py

```diff
@@ -769,10 +769,11 @@ def sai_thrift_read_pg_drop_counters(client, port_id):
 
     return pg_cntrs
 
-def sai_thrift_read_pg_shared_watermark(client, port_id):
-    pg_cntr_ids=[
-        SAI_INGRESS_PRIORITY_GROUP_STAT_SHARED_WATERMARK_BYTES
-    ]
+def sai_thrift_read_pg_shared_watermark(client, asic_type, port_id):
+    if 'cisco-8000' in asic_type:
+        pg_cntr_ids = [SAI_INGRESS_PRIORITY_GROUP_STAT_CURR_OCCUPANCY_BYTES]
```

Comment:
So looks like it is just the current queue stats? If so, how it works as watermark?